editors/sc-ide/widgets/main_window.cpp

```diff
@@ -67,6 +67,7 @@
 #include <QUrl>
 #include <QMimeData>
 #include <QMetaMethod>
+#include <QDebug>
```

Comment:
This is not directly related but it was missing.. Let me know if I should revert that one.

Reply:
if you included it for `qDebug()`, it's not really necessary as `qDebug` is a global feature: http://doc.qt.io/qt-5/qtglobal.html#qDebug